tests/unit/responses/elsevier/j.nima.2019.162728.xml

```diff
@@ -0,0 +1 @@
d="b7"><ce:label>[7]</ce:label><ce:other-ref id="sb7"><ce:textref><ce:inter-ref id="interref2" xlink:href="http://afi.jinr.ru" xlink:type="simple">http://afi.jinr.ru</ce:inter-ref>.</ce:textref></ce:other-ref></ce:bib-reference><ce:bib-reference id="b8"><ce:label>[8]</ce:label><sb:reference id="sb8"><sb:contribution langtype="en"><sb:authors><sb:author><ce:surname>Mikhaylov</ce:surname><ce:given-name>V.</ce:given-name></sb:author><sb:et-al/></sb:authors></sb:contribution><sb:host><sb:issue><sb:series><sb:title><sb:maintitle>Nucl. Instrum. Meth.</sb:maintitle></sb:title><sb:volume-nr>A912</sb:volume-nr></sb:series><sb:date>2018</sb:date></sb:issue><sb:pages><sb:first-page>241</sb:first-page><sb:last-page>244</sb:last-page></sb:pages></sb:host></sb:reference><ce:source-text id="afs8">V. Mikhaylov et al., Nucl.Instrum.Meth. A912 (2018) 241-244</ce:source-text></ce:bib-reference><ce:bib-reference id="b9"><ce:label>[9]</ce:label><ce:other-ref id="sb9"><ce:textref><ce:inter-ref id="interref3" xlink:href="http://dpnc.unige.ch/bravar/exp/NA61/Bravar-DRS.pdf" xlink:type="simple">http://dpnc.unige.ch/bravar/exp/NA61/Bravar-DRS.pdf</ce:inter-ref>.</ce:textref></ce:other-ref></ce:bib-reference><ce:bib-reference id="b10"><ce:label>[10]</ce:label><ce:other-ref id="sb10"><ce:textref>O. Gavrishchuk, et al. <ce:inter-ref id="interref4" xlink:role="http://www.elsevier.com/xml/linking-roles/preprint" xlink:href="arxiv:1406.1599v2[physics.ins-det]" xlink:type="simple">arXiv:1406.1599v2[physics.ins-det]</ce:inter-ref>, 2014.</ce:textref></ce:other-ref></ce:bib-reference><ce:bib-reference id="b11"><ce:label>[11]</ce:label><sb:reference id="sb11"><sb:contribution langtype="en"><sb:authors><sb:author><ce:surname>Guber</ce:surname><ce:given-name>F.</ce:given-name></sb:author><sb:et-al/></sb:authors></sb:contribution><sb:host><sb:issue><sb:series><sb:title><sb:maintitle>EPJ Web Conf.</sb:maintitle></sb:title><sb:volume-nr>204</sb:volume-nr></sb:series><sb:date>2019</sb:date></sb:issue><sb:pages><sb:first-page>07007</sb:first-page></sb:pages><ce:doi>10.1051/epjconf/201920407007</ce:doi></sb:host></sb:reference><ce:source-text id="afs11">F. Guber et al., submitted to EPJ Web Conf. www.epj-conferences.org</ce:source-text></ce:bib-reference><ce:bib-reference id="b12"><ce:label>[12]</ce:label><ce:other-ref id="sb12"><ce:textref><ce:inter-ref id="interref5" xlink:href="https://ieeexplore.ieee.org/document/5402172" xlink:type="simple">https://ieeexplore.ieee.org/document/5402172</ce:inter-ref>.</ce:textref></ce:other-ref></ce:bib-reference><ce:bib-reference id="b13"><ce:label>[13]</ce:label><ce:other-ref id="sb13"><ce:textref>F. Guber, et al. <ce:inter-ref id="interref6" xlink:href="http://repository.gsi.de/record/109059/files/20150720-CBM-TDR-PSD.pdf" xlink:type="simple">http://repository.gsi.de/record/109059/files/20150720-CBM-TDR-PSD.pdf</ce:inter-ref>.</ce:textref></ce:other-ref></ce:bib-reference></ce:bibliography-sec></ce:bibliography></ja:tail></ja:article></doc:document>
```

Comment:
I don't think we're allowed to put this in a public repo. The main problem is the full content of the article being available. Can you replace the full-text by some dummy text (like `Full-text removed for copyright reasons`), keeping the structure?